.github/workflows/generic_repodispatch.yml

```diff
@@ -2,7 +2,7 @@ name: generic-repodispatch
 
 on:
   repository_dispatch:
-    types: [run-status-changed]
+    types: [model-registered2]
```

Comment:
model-registered2 is not any event

Reply:
no this is not a event.Changed the name so that this workflow does not gets trigger.